From fd50abb2bcfe64b247363d3716b92efa3e22ee12 Mon Sep 17 00:00:00 2001
Message-ID: <fd50abb2bcfe64b247363d3716b92efa3e22ee12.1740173176.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1740173176.git.sharlatanus@gmail.com>
References: <cover.1740173176.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Mon, 17 Feb 2025 17:01:09 +0000
Subject: [PATCH 25/51] gnu: python-ndcube: Update to 2.3.1.

* gnu/packages/astronomy.scm (python-ndcube): Update to 2.3.1.
[native-inputs]: Remove python-setuptools-scm; add python-setuptools-scm-next.

Change-Id: Ib8472b7ce9a7d52554304078ad7e397ff8c68d96
---
 gnu/packages/astronomy.scm | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 3147674ae89..1d252cc6f45 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -4478,13 +4478,13 @@ (define-public python-naima
 (define-public python-ndcube
   (package
     (name "python-ndcube")
-    (version "2.3.0")
+    (version "2.3.1")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "ndcube" version))
        (sha256
-        (base32 "1pza5g38gn941dwy8yahkd37lb7z80czs1yj38xhn2cf0s69779h"))))
+        (base32 "0hf261l36jxmd0r8sh2vyxaapv8gcdmnnv82k0hqvq6wh1qp5ksv"))))
     (build-system pyproject-build-system)
     (arguments
      (list
@@ -4526,7 +4526,7 @@ (define-public python-ndcube
            python-pytest-xdist
            python-scipy
            python-setuptools
-           python-setuptools-scm
+           python-setuptools-scm-next
            python-sunpy
            python-wheel))
     (home-page "https://docs.sunpy.org/projects/ndcube/")
-- 
2.47.1

